src/publisher.ts

```diff
@@ -17,19 +17,16 @@
 import {promisifyAll} from '@google-cloud/promisify';
 import * as arrify from 'arrify';
 import {CallOptions} from 'google-gax';
-import {ServiceError} from 'grpc';
```

Comment:
Why are we removing this? Seems like if an error occurred we would want the user to be able to inspect the status code/metadata.